portfolio/src/main/webapp/index.html

```diff
@@ -28,6 +28,28 @@ <h2>My Background</h2>
           </p>
       <h2>My Pets</h2>
       <p>I love my pets and I hope you enjoy their portraits below:</p>
+      <div id="pets">
+          <div class="row">
+            <div class="column">
+                <img src="/images/Fern.jpeg" alt="Fern" style="width:100%">
```

Comment:
Optional: I think you can replace this inline style with a style on img children of the "column" div.

Example:
https://jsfiddle.net/8ksL2gx4/